philips-hue-adapter.js

```diff
@@ -365,8 +380,21 @@ class PhilipsHueDevice extends Device {
                     type: 'boolean',
                     readOnly: true,
                   },
-                  device.state.buttonevent >= buttonInfo.start &&
-                                       device.state.buttonevent < buttonInfo.off
+                  device.state.buttonevent >= buttonInfo.initialPress &&
+                    device.state.buttonevent <= buttonInfo.longRelease
+                )
+              );
+              this.properties.set(
+                'lastUpdated',
+                new PhilipsHueProperty(
+                  this,
+                  buttonType,
+                  {
+                    label: buttonInfo.label,
```

Comment:
This should have its own label, maybe something like:
```js
`${buttonInfo.label} (Last Updated)`
```

Reply:
I've modified the property to be on the switch rather than on the button. This now has the added benefit of displaying the lastUpdated property in the switch view.

![Screenshot from 2020-07-10 21-17-36](https://user-images.githubusercontent.com/33864869/87199070-e3539000-c2f2-11ea-81cc-885072467925.png)